interview.md

```diff
@@ -58,11 +78,17 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I belive most of this project would be spend on the development of the language  and less so on the development of the system. This project whould have to optimize user understanging of the language as well as user speed, and language difficulty. 
+
 ### Scope
 
 _How big an idea is this? How ambitious is this project?_
 
+I feel like this project is something that could be done within a smaller time span such as a semester I also feel like the uses for such could be very big within the poker world. 
+
 ### Benefits and drawbacks
 
 _Why might this be a good idea for a project? Why might this not be a good idea
 project?_
+
+I feel like this is a cool project to work on because it peaks my interst in poker, with that being said there are some negative connotations around poker and the class may not decide this is the best use for a DSL. However I think there are many aspects of this that would be interesting to intagrate such as an optimal speed DSL for poker players, a way to save and analyze a natural like langauge, and automation of data structures and or databases. 
```

Comment:
I think it's a very interesting use case for a DSL personally, though I do wonder (like I said in my first comment) how many official poker and gambling spaces would embrace such a tool. For more advanced amateurs though, I can imagine this could be really useful.